modules/cart/js/commerce_cart.js

```diff
@@ -8,19 +8,30 @@
 
   Drupal.behaviors.commerceCartBlock = {
     attach: function (context) {
-      var $context = $(context);
-      var $cart = $context.find('.cart--cart-block');
-      var $cartContents = $cart.find('.cart-block--contents');
+      var $context = $(context),
+          $cart = $context.find('.cart--cart-block'),
+          $cartButton = $context.find('.cart-block--link__expand'),
+          $cartContents = $cart.find('.cart-block--contents');
 
       if ($cartContents.length > 0) {
-        $cart.click(function (e) {
+        // Expand the block when the link is clicked.
+        $cartButton.on('click', function (e) {
+          // Prevent it from going to the cart.
           e.preventDefault();
+          // Get the shopping cart width + the offset to the left.
+          var $windowWidth = $(window).width(),
```

Comment:
As a general nitpick, I only place `$` on variables if they are a jQuery object. The variables defined in this section are all integers, so we can just use `windowWidth`, `cartOffsetLeft`, `cartWidth` respectively.